toonz/sources/toonz/menubar.cpp

```diff
@@ -1433,6 +1433,7 @@ QMenuBar *StackedMenuBar::createFullMenuBar() {
   addMenuItem(helpMenu, MI_OpenOnlineManual);
   helpMenu->addSeparator();
   addMenuItem(helpMenu, MI_About);
+  addMenuItem(helpMenu, MI_ClearCacheFolder);
```

Comment:
In `menubar_template.xml`, you have `Clear Cache Folder` above `About`, but reversed here.   I would put it above `About` and add a separator between them. Same in the template.

Seems like a rather odd placement for this command. I'd probably put this under the `File` menu myself.

Reply:
Thank you @manongjohn for reviewing this.
Sure, I'll move the command under the `File` menu.